src/cts/test/balance_levels.ok

```diff
@@ -39,30 +39,22 @@ TritonCTS forced slew degradation on these wires.
     Sinks per sub-region: 76
     Sub-region size: 6.7460 X 6.3491
 [INFO CTS-0034]     Segment length (rounded): 4.
-    Key: 14 inSlew: 1 inCap: 1 outSlew: 2 load: 1 length: 4 delay: 1
 [WARNING CTS-0045] Creating fake entries in the LUT.
  Level 2
     Direction: Vertical
     Sinks per sub-region: 38
     Sub-region size: 6.7460 X 3.1746
 [INFO CTS-0034]     Segment length (rounded): 1.
-    Key: 882 inSlew: 1 inCap: 1 outSlew: 2 load: 1 length: 1 delay: 1
  Level 3
     Direction: Horizontal
     Sinks per sub-region: 19
     Sub-region size: 3.3730 X 3.1746
 [INFO CTS-0034]     Segment length (rounded): 1.
-    Key: 882 inSlew: 1 inCap: 1 outSlew: 2 load: 1 length: 1 delay: 1
  Level 4
```

Comment:
@precisionmoon I mean the messages below. Are they useful to the user?

```
Level 4
    Direction: Vertical
    Sinks per sub-region: 10
    Sub-region size: 3.3730 X 1.5873
[INFO CTS-0034]     Segment length (rounded): 1
```

Reply:
Yes, these are helpful in understanding how HTrees are constructed.   It gives information about direction, # sinks, region size and intended segment length.  For example, if sinks per region is 10 at level 4, you want to check if all buffers at level 4 are driving 10 loads.  If not, you can try inserting dummy loads (-use_dummy_load) to balance tree loads.  Segment length should decrease as level increases; otherwise, this is an issue that needs to be looked at.  Also, we want to preserve ideal segment length even in the presence of macros or blockages.   Does this make sense?

Reply:
I dont know enough about the domain here... Is this information in the docs or are these things that an experienced engineer in EDA tools would know?